src/Command/DocsCommand.php

```diff
@@ -26,7 +31,82 @@ protected function configure() {
    * @return int
    */
   protected function execute(InputInterface $input, OutputInterface $output) {
-    $this->localMachineHelper->startBrowser('https://docs.acquia.com/acquia-cli/');
+    $acquia_products = [
```

Comment:
Looks like we're still missing those products:

* BLT
* Cloud Platform
* Commerce Manager
* Lando
* Acquia DAM
* Acquia DAM Classic
* Personalization
* Search
* Campaign Factory
* Campaign Studio
* Platform APIs
* Remote Administration

Reply:
`Campaign Studio` is already in.
`Search` is already in. 
`Commerce Manager` I didn;t add as this is because AFAIK we now don;t support/suggest commerce manager.
`Lando` Not added as lando is not acquia product + on lando page https://docs.acquia.com/lando/ we recommend to use cloud IDE

`Platform APIs` and `Remote Administration` - couldn't find link/info on https://docs.acquia.com/

PR updated for other products